### PR TITLE
Fixed circular dependencis;

### PR DESCRIPTION
**Pull request by richardspence**
Updated UMD template to consume require and exports.

**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**If relevant, link to documentation update:**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by richardspence** [comment]
#4562

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,3 @@
 .DS_Store
 *.log
 .idea
-.vscode
```
Comment (bebraw) [inline .gitignore]: No need for this change.

**Comment by bebraw** [comment]
Can you sign the CLA?

File: .gitignore (hunk 1 above)
Comment (richardspence) [inline .gitignore]: I'd thought I'd check in the vscode project settings so future contributors using vscode would have the correct style settings for this repro.  It shouldn't do any harm having those settings checked in.

Comment (bebraw) [inline .gitignore]: Ah, ok. Maybe separate PR but I get it. Thanks.

**Comment by sokra** [comment]
closing, because it's old and discontinued

**Pull request closed without merging**